CHANGELOG.md

```diff
@@ -0,0 +1,29 @@
+# Changelog
+All notable changes to this crate will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [2.0.0] - 2021-01-26
+
+### Added
+- `Decode::skip` is introduced, allowing to skip some encoded types.
```

Comment:
I'd remove the "is introduced", we're already under the `### Added` heading, so:

"`Decode::skip` allows selectively skipping some types when decoding (PR #201)" maybe?